manifests/install.pp

```diff
@@ -98,9 +98,15 @@
           }
 
           if ( $::grafana::manage_package_repo ){
+            # http://docs.grafana.org/installation/rpm/#install-via-yum-repository
+            $baseurl = $::grafana::repo_name ? {
```

Comment:
please don't add leading `::` after the `$`. That is deprecated syntax that was required for puppet 3.

Reply:
Yes, I'm aware of that, I was just being consistent with the rest of the file. Will change.